app/controllers/v0/virtual_agent/virtual_agent_claim_controller.rb

```diff
@@ -32,19 +56,27 @@ def index
         end
       end
 
-      def show
-        claim = EVSSClaim.for_user(current_user).find_by(evss_id: params[:id])
-
-        claim, synchronized = service.update_from_remote(claim)
+      def poll_claims_from_lighthouse
+        lighthouse_client_id = Settings.lighthouse.benefits_claims.access_token.client_id
```

Comment:
You don't need to pass the client_id or rsa_key_path if you are using the ones from `Settings.lighthouse.benefits_claims.access_token`. The service will use these by default if neither is provided

Reply:
We addressed your concern. Please confirm.